test/test-functions

```diff
@@ -2003,7 +2006,7 @@ test_setup() {
         mount_initdir
     elif [ -e "$public" ]; then
         echo "Reusing existing cached image $PWD/$public → $(realpath $public)"
-        ln -s "$(realpath $public)" "$image"
+        cp -v "$(realpath $public)" "$image"
```

Comment:
So... I'm looking at `qemu-img create -B backing_file`. We could avoid the copy I think.

Reply:
That sounds promising! Wouldn't it still be issue for nspawn, though, since it uses the *base* image as well?